test/e2e/placement_cro_test.go

```diff
@@ -144,9 +143,25 @@ var _ = Describe("creating clusterResourceOverride (selecting all clusters) to o
 		want := map[string]string{croTestAnnotationKey: croTestAnnotationValue1}
 		checkIfOverrideAnnotationsOnAllMemberClusters(true, want)
 	})
+
+	It("delete the cro attached to this CRP", func() {
```

Comment:
do you want to move to the AfterAll to make it consistent with ro test?

Reply:
I kept it this way to try both types. The original goal is to add that in all BeforeAll but I can't put all the context inside one describe.